packages/graphql-utils/package.json

```diff
@@ -28,9 +28,5 @@
     "typescript": "^4.2.4"
   },
   "dependencies": {
-    "@babel/traverse": "^7.15.4",
-    "@babel/types": "^7.15.6",
-    "@graphql-codegen/plugin-helpers": "^2.2.0",
-    "@graphql-tools/relay-operation-optimizer": "^6.4.0"
```

Comment:
Removing to speed things up a bit when installing faststore packages and avoid having junk on node_modules.